spec.html

```diff
@@ -20501,7 +20507,7 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. If |LeftHandSideExpression| is neither an |ObjectLiteral| nor an |ArrayLiteral|, then
           1. Let _lref_ be ? Evaluation of |LeftHandSideExpression|.
           1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) and IsIdentifierRef of |LeftHandSideExpression| are both *true*, then
-            1. Let _rval_ be ? NamedEvaluation of |AssignmentExpression| with argument _lref_.[[ReferencedName]].
+            1. Let _rval_ be ? NamedEvaluation of |AssignmentExpression| with argument StringValue of |LeftHandSideExpression|.
```

Comment:
Nit: Here and below, this reads a bit awkward given the invocation of the SDO. Perhaps pull out a separate step like `Let _lhs_ be the StringValue of |LeftHandSideExpression|.`

Very weak preference.

Reply:
Related: we already wanted to do this for [other reasons](https://github.com/tc39/ecma262/pull/3309#issuecomment-2071149881). 👍

Reply:
Fixed.